src/libs/elektra/keyset.c

```diff
@@ -576,6 +576,11 @@ Key *k2 = keyNew("user:/b", KEY_END);
  * Do not strcmp the keyName() yourself, because
  * the result differs from simple ascii comparison.
  *
+ * @pre The Keys @p k1 and @p k2 have been properly initialized via keyNew() or are NULL
+ * @invariant All parts of the Keys remain unchanged
```

Comment:
What about ref counters?

Reply:
Doesn't look like `keyCmp ()` changes the refcounters.